src/api/ingest/serializers.py

```diff
@@ -12,6 +12,12 @@
 from ..resources.choices import ChoiceViewSet
 
 
+__all__ = [
```

Comment:
Again I may benefit from having this explained... how does this file know about the classes in its sister files?